spec/helpers/application_helper_spec.rb

```diff
@@ -2,9 +2,14 @@
 
 RSpec.describe ApplicationHelper, type: :helper do
   describe "#current_user" do
+    session_user = {
+      name: "Test User",
+      email: "testuser@example.com"
+    }
```

Comment:
I'd generally use a `let` block here in rspec. Is there an advantage to just creating a bare variable in this scope?

Reply:
i don't think so, i'll update it to a `let`!